Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -244,14 +244,44 @@ theorem stabilizer_smul_eq_stabilizer_map_conj (g : G) (a : α) :
   rw [mem_stabilizer_iff, ← smul_left_cancel_iff g⁻¹, smul_smul, smul_smul, smul_smul,
     inv_mul_cancel, one_smul, ← mem_stabilizer_iff, Subgroup.mem_map_equiv, MulAut.conj_symm_apply]
 
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer {a b : α} {g : G} (hg : g • b = a) :
+    stabilizer G b ≃* stabilizer G a :=
```

Comment:
I wonder if it makes more sense to state this as `stabilizer G b ≃* stabilizer G (g • b)` (or the reverse), that way you don't need to take a propositional argument to this. And we could have `copy : stabilizer G a ≃* stabilizer G b` to ease translation, as is the usual pattern. But let's perhaps also see what other reviewers think about this.

Reply:
I tried to do that, the definitions are OK, but then the trans property is unnatural: to be stated, it needs to rewrite `h • g • a = h * g • a`,  and then I can't prove that it holds.

Reply:
I've asked about this in [#mathlib4 > avoiding HEq @ 💬](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/avoiding.20HEq/near/512266402)